about.md

```diff
@@ -18,5 +18,6 @@ You can improve the community’s shared knowledge by [contributing missing patt
 If you have feedback about the catalogue, then email [hello@projectsbyif.com](mailto:hello@projectsbyif.com).
 
 <div class="iframe-container">
-  <iframe src="https://player.vimeo.com/video/231723090" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+  <!-- Removed frameborder RE: deprecated in HTML5 -->
```

Comment:
This feels like something that is ok to describe only in the commit message rather than as an HTML comment unless you think there is benefit in keeping it here in case this embed code is updated in the future